README.md

```diff
@@ -30,8 +30,5 @@ The following set of features will be required to support the real-world use-cas
    * Adapters can be created for devices that do not directly support gNMI/gNOI interfaces. Such adapters can be deployed either as proxy agents or potentially can be hosted on the devices themselves.
 
 ## Additional Documentation
-* [How to contribute][contributing] to µONOS
-* [How to run][running] µONOS
-
-[contributing]: https://github.com/opennetworkinglab/onos-config/blob/master/docs/CONTRIBUTING.md
-[running]:  https://github.com/opennetworkinglab/onos-config/blob/master/docs/RUN.md
+* [How to contribute](./docs/CONTRIBUTING.md) to µONOS-config
```

Comment:
I wonder if we should just stick with calling it onos-config to address the repo and "ONOS Configuration" to address the subsystem/subproject. I'd leave µONOS for just a concise code-name for the new architecture in slides and such.